code/modules/mob/living/carbon/superior_animal/roach/roach_summon.dm

```diff
@@ -0,0 +1,66 @@
+// A one-use item that summon a specific type of roach and tame it.
+/* Here is the current list of choices :
+	- "kampfer" : Default Roach
+	- "unbekannt" : Bluespace Roach
+	- "elektromagnetisch" : EMP Roach
+	- "fuhrer" : Fuhrer Roach
+	- "gluhend" : Glowing Roach
+	- "jager" : Hunter Roach
+	- "kaiser" : Kaiser Roach
+	- "kraftwerk" : Nanite Roach
+	- "seuche" : Support Roach
+	- "panzer" : Tank Roach
+	- "gestrahlte" : Toxic Roach
+*/
+/obj/item/roach_summon
+	name = "bluespace roach beacon"
```

Comment:
Rather just use the cubes

Reply:
Cubes don't tame the roaches. This does.
Also this is meant to be unique/admin only.